src/components/App.js

```diff
@@ -12,6 +12,21 @@ const Root = styled.div`
   height: 100vh;
 `
 
+var now = new Date()
+var millisecondsUntil5 =
+  new Date(now.getFullYear(), now.getMonth(), now.getDate(), 5, 0, 0, 0) - now
+if (millisecondsUntil5 < 0) {
+  millisecondsUntil5 += 86400000 // it after 5am, try again at 5am tomorrow.
+}
+
+setTimeout(() => {
+  location.href = '/'
+  if (location.href === '/') {
+    //The if statement stops other pages reloading before redirecting
+    location.reload()
+  }
+}, millisecondsUntil5)
```

Comment:
This might look better as  funcitonal React component where you do the `setTimeout` in a `useEffect`. That way you can also `useNavigate` rather than messing with the low level stuff. Just a suggestion

Reply:
Agreed